pkg/operator/sync_common.go

```diff
@@ -28,145 +28,195 @@ func manageControllerNS(c serviceCAOperator) (bool, error) {
 	return modified, err
 }
 
-func manageSignerControllerResources(c serviceCAOperator) (bool, string) {
-	return manageControllerResources(c, "v4.0.0/service-serving-cert-signer-controller/")
+func manageSignerControllerResources(c serviceCAOperator, modified *bool) error {
+	return manageControllerResources(c, "v4.0.0/service-serving-cert-signer-controller/", modified)
 }
 
-func manageAPIServiceControllerResources(c serviceCAOperator) (bool, string) {
-	return manageControllerResources(c, "v4.0.0/apiservice-cabundle-controller/")
+func manageAPIServiceControllerResources(c serviceCAOperator, modified *bool) error {
+	return manageControllerResources(c, "v4.0.0/apiservice-cabundle-controller/", modified)
 }
 
-func manageConfigMapCABundleControllerResources(c serviceCAOperator) (bool, string) {
-	return manageControllerResources(c, "v4.0.0/configmap-cabundle-controller/")
+func manageConfigMapCABundleControllerResources(c serviceCAOperator, modified *bool) error {
+	return manageControllerResources(c, "v4.0.0/configmap-cabundle-controller/", modified)
 }
 
-func manageControllerResources(c serviceCAOperator, resourcePath string) (bool, string) {
+func manageControllerResources(c serviceCAOperator, resourcePath string, modified *bool) error {
```

Comment:
I quite prefer returning the `modified` flag out rather than having a bool pointer being passed down as an input argument.